ddtrace/contrib/asgi/middleware.py

```diff
@@ -70,13 +70,12 @@ def _default_handle_exception_span(exc, span):
 def span_from_scope(scope):
     # type: (Mapping[str, Any]) -> Optional[Span]
     """Retrieve the top-level ASGI span from the scope."""
-    return scope.get("datadog", {}).get("request_span")
+    return scope.get("datadog", {}).get("request_spans", [{}])[0]
```

Comment:
this is not correct, it will return an empty dict if no request spans are present, we need it to return None instead.

```suggestion
    return scope.get("datadog", {}).get("request_spans", [None])[0]
```

But I don't think I like mixing types like this, since `request_spans` is supposed to be a `List[Span]` and this treats it like a `List[Optional[Span]]` ?